src/proxy/http2/Http2ConnectionState.cc

```diff
@@ -993,6 +1005,18 @@ Http2ConnectionState::rcv_continuation_frame(const Http2Frame &frame)
                       "continuation bad client id");
   }
 
+  if (payload_length == 0 && (frame.header().flags & HTTP2_FLAGS_HEADERS_END_HEADERS) == 0x0) {
+    this->increment_received_empty_frame_count();
+    if (configured_max_empty_frames_per_minute >= 0 &&
+        this->get_received_empty_frame_count() > static_cast<uint32_t>(configured_max_empty_frames_per_minute)) {
+      Metrics::Counter::increment(http2_rsb.max_empty_frames_per_minute_exceeded);
+      Http2StreamDebug(this->session, stream_id, "Observed too frequent empty frames: %u within a last minute",
```

Comment:
Being explicit about these being CONTINUATION frames might be helpful to the reader:

```
Observed too many empty CONTINUATION frames: %u within the last minute
```